src/frontend/context/SimuleringContext.tsx

```diff
@@ -100,23 +100,10 @@ const [SimuleringProvider, useSimulering] = constate(({ åpenBehandling }: IProp
 
     const erMigreringFraInfotrygd = åpenBehandling.type === Behandlingstype.MIGRERING_FRA_INFOTRYGD;
 
-    const skalStoppeISimulering = () => {
-        if (åpenBehandling.årsak === BehandlingÅrsak.HELMANUELL_MIGRERING && simResultat) {
-            const tidligereUtbetaltPerioderEtterbetalingOver220 = simPerioderFørMars2023.filter(
-                periode => periode.etterbetaling && periode.etterbetaling > 220
-            );
-            return erFeilutbetaling || tidligereUtbetaltPerioderEtterbetalingOver220.length > 0;
-        }
-        return erFeilutbetaling || erEtterutbetaling;
-    };
-    const erMigreringMedStoppISimulering = erMigreringFraInfotrygd && skalStoppeISimulering();
+    const erAvvikISimuleringForBehandling = erFeilutbetaling || erEtterutbetaling;
 
-    const harKunNegativeEllerKunPositiveAvvik = (perioderesultater: number[]) => {
```

Comment:
Nå husker jeg litt bedre hvor denne kom fra: Den gjorde den samme sjekken som funksjonene `harKunPositiveResultater` og `harKunNegativeResultater` gjør i backend, bare slått sammen til èn funksjon felles for både feilutbetaling og etterbetaling